From d5957578743efcc9fbdab936c60ada43f9a29098 Mon Sep 17 00:00:00 2001
From: fundamental <mark.d.mccurry@gmail.com>
Date: Wed, 23 Sep 2015 14:03:19 -0400
Subject: [PATCH 0538/1936] Resolve Licensing Ambiguity

GPL v2 -> GPL v2+ based upon rereading commit logs and the mailing
lists
---
 README.adoc                    | 2 +-
 src/Misc/WavFile.cpp           | 2 +-
 src/Misc/WavFile.h             | 2 +-
 src/Nio/WavEngine.cpp          | 2 +-
 src/Nio/WavEngine.h            | 2 +-
 src/Output/DSSIaudiooutput.cpp | 2 +-
 src/main.cpp                   | 2 +-
 7 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/README.adoc b/README.adoc
index 25be6a1b..80c8bf06 100644
--- a/README.adoc
+++ b/README.adoc
@@ -8,7 +8,7 @@ Originally written by Nasca Octavian Paul <zynaddsubfx AT yahoo D0T com>
 Maintained by Mark McCurry 2009+.
 
 ZynAddSubFX is free program and is distributed WITH NO WARRANTY.
-It is licensed under GNU General Public License version 2 (and only version 2).
+It is licensed under GNU General Public License version 2+.
 
 The project page is
 http://sourceforge.net/projects/zynaddsubfx
diff --git a/src/Misc/WavFile.cpp b/src/Misc/WavFile.cpp
index 78db9d52..43d66554 100644
--- a/src/Misc/WavFile.cpp
+++ b/src/Misc/WavFile.cpp
@@ -10,7 +10,7 @@
   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-  GNU General Public License (version 2) for more details.
+  GNU General Public License (version 2 or later) for more details.
 
   You should have received a copy of the GNU General Public License (version 2)
   along with this program; if not, write to the Free Software Foundation,
diff --git a/src/Misc/WavFile.h b/src/Misc/WavFile.h
index 4b29efa0..b33a75f0 100644
--- a/src/Misc/WavFile.h
+++ b/src/Misc/WavFile.h
@@ -13,7 +13,7 @@
   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-  GNU General Public License (version 2) for more details.
+  GNU General Public License (version 2 or later) for more details.
 
   You should have received a copy of the GNU General Public License (version 2)
   along with this program; if not, write to the Free Software Foundation,
diff --git a/src/Nio/WavEngine.cpp b/src/Nio/WavEngine.cpp
index efc899bf..ef54c85b 100644
--- a/src/Nio/WavEngine.cpp
+++ b/src/Nio/WavEngine.cpp
@@ -9,7 +9,7 @@
   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-  GNU General Public License (version 2) for more details.
+  GNU General Public License (version 2 or later) for more details.
 
   You should have received a copy of the GNU General Public License (version 2)
   along with this program; if not, write to the Free Software Foundation,
diff --git a/src/Nio/WavEngine.h b/src/Nio/WavEngine.h
index 6ec6d743..aeb26394 100644
--- a/src/Nio/WavEngine.h
+++ b/src/Nio/WavEngine.h
@@ -13,7 +13,7 @@
   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-  GNU General Public License (version 2) for more details.
+  GNU General Public License (version 2 or later) for more details.
 
   You should have received a copy of the GNU General Public License (version 2)
   along with this program; if not, write to the Free Software Foundation,
diff --git a/src/Output/DSSIaudiooutput.cpp b/src/Output/DSSIaudiooutput.cpp
index fef0f9ff..d4229fd7 100644
--- a/src/Output/DSSIaudiooutput.cpp
+++ b/src/Output/DSSIaudiooutput.cpp
@@ -534,7 +534,7 @@ DSSI_Descriptor *DSSIaudiooutput::initDssiDescriptor()
             newLadspaDescriptor->Name  = "ZynAddSubFX";
             newLadspaDescriptor->Maker =
                 "Nasca Octavian Paul <zynaddsubfx@yahoo.com>";
-            newLadspaDescriptor->Copyright = "GNU General Public License v2";
+            newLadspaDescriptor->Copyright = "GNU General Public License v2 or later";
             newLadspaDescriptor->PortCount = 2;
 
             newPortNames    = new const char *[newLadspaDescriptor->PortCount];
diff --git a/src/main.cpp b/src/main.cpp
index eab03bbc..d495cf06 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -138,7 +138,7 @@ int main(int argc, char *argv[])
     << "                Copyright (c) 2009-2014 Mark McCurry [active maintainer]"
     << endl;
     cerr << "Compiled: " << __DATE__ << " " << __TIME__ << endl;
-    cerr << "This program is free software (GNU GPL v.2 or later) and \n";
+    cerr << "This program is free software (GNU GPL v2 or later) and \n";
     cerr << "it comes with ABSOLUTELY NO WARRANTY.\n" << endl;
     if(argc == 1)
         cerr << "Try 'zynaddsubfx --help' for command-line options." << endl;
-- 
2.47.0

